docs/changelog.md

```diff
@@ -548,6 +548,9 @@ what we publish.
 - `String.split()` now defaults to whitespace and has pythonic behavior in that
   it removes all adjacent whitespaces by default.
 
+- `String` now has a `String.__iter__()` method which returns a `StringRef`
```

Comment:
Is it possible to use `StringSlice` instead of `StringRef` for improved safety? We're hoping to remove most uses of `StringRef` in favor of safer alternatives.